meetings/2019/2019-01-14.md

```diff
@@ -30,23 +30,22 @@ Extracted from **bootstrap-agenda** labelled issues and pull requests from the *
 * CPC Charter
   - 5 minute timebox
   - Refs:
-    - [Proposal Readme](https://github.com/nodejs/bootstrap/blob/master/proposals/mdawson-cpc-charter/README.md)
+    - [Proposal Readme](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/proposals/approved/CPC_CHARTER/README.md)
     - https://github.com/nodejs/bootstrap/pull/65
 
 Mdawson: Update: 2 prs since last discussion. Board represntation wasn’t in the first pr. The proposal includes 1 board seat for the chair as well as 2 other seats tbd by the CPC. For the first year those proceses would be 1) dedicated to the Node projects and the other being a vote of the members with candidates of non-node projects. The folloing years the CPC could decide. Otehr comments have been addressed. Make it clearer that anyone can show up to participate so we added ‘Observers’. Distinction that regular members make contributions on an ongoing basis. Other comments can be discussed later before we progress too much. 
 
 * Project Progression
   - 5 minute timebox
   - Refs:
-    - [Proposal Readme](https://github.com/nodejs/bootstrap/blob/master/proposals/jorydotcom-PROJECT_PROGRESSION/README.md)
+    - [Proposal Readme](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/proposals/approved/PROJECT_PROGRESSION/README.md)
 
 Jory: Great meeting this morning, Action items to define open governance for top level projects and to fill gap around code of conduct expectations for projects. Good basis of progress from which to move forward.
 
 * Expectations
   - 5 minute timebox
   - Refs:
-    - [Expectations.md](https://github.com/nodejs/bootstrap/blob/master/proposals/hackygolucky-EXP
-ECTATIONS/EXPECTATIONS.md)
+    - [Expectations.md](https://github.com/openjs-foundation/cross-project-council/blob/main/proposals/approved/EXPECTATIONS/EXPECTATIONS.md)
```

Comment:
```suggestion
    - [Expectations.md](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/proposals/approved/EXPECTATIONS/EXPECTATIONS.md)
```
sorry there's so many of these :-/ (there's more, but i'll only comment once)

Reply:
> sorry there's so many of these :-/ (there's more, but i'll only comment once)

no worries! I appreciate the work <3